src/object/level_time.hpp

```diff
@@ -52,7 +52,7 @@ class LevelTime final : public GameObject,
 
   /** @} */
   virtual std::string get_class() const override { return "leveltime"; }
-  virtual std::string get_display_name() const override { return _("Level time"); }
+  virtual std::string get_display_name() const override { return _("Timer"); }
```

Comment:
Maybe `Time Limit` (LevelTime is the time limit and not the target time, I think).